src/data/nat/factorization.lean

```diff
@@ -255,6 +255,10 @@ lemma prime.pow_dvd_iff_dvd_pow_factorization {p k n : ℕ} (pp : prime p) (hn :
   p ^ k ∣ n ↔ p ^ k ∣ p ^ n.factorization p :=
 by rw [pow_dvd_pow_iff_le_right pp.one_lt, pp.pow_dvd_iff_le_factorization hn]
 
+lemma prime.dvd_iff_one_le_factorization {p n : ℕ} (pp : prime p) (hn : n ≠ 0) :
```

Comment:
is it worth making a backwards-only version of this that doesn't require `pp`? just for simplicity's sake

Reply:
Added, further up the file!